api/openapi/api.oas2.yml

```diff
@@ -1030,7 +1033,7 @@ paths:
           name: body
           schema:
             $ref: '#/definitions/address-book-input'
-      description: Updates a user's address book.
+      description: "Updates a user's address book.\n\n**Note**: if the passed `id` matches an existing `address` a new `Spree::Address` record will be created and the matched `address` `archived` on `Spree::UserAddress`. For a similar logic, if the passed `id` matches an existing `address` which is in `archived` state, the `Spree::UserAddress` record will be restored: `archived: false`. \n\nSee `user_address_book.rb` for further information."
```

Comment:
Why not using `|-` as well here?

Reply:
Hey @kennyadsl, 
fixed that description! https://github.com/solidusio/solidus/pull/3509/files#diff-a84e9b384a854939d2dc44eb24497283R1037

Now it is consistent with the rest of the text.